boot/bootutil/include/bootutil/image.h

```diff
@@ -37,12 +37,7 @@ struct flash_area;
  * Image header flags.
  */
 #define IMAGE_F_PIC                      0x00000001 /* Not supported. */
-#define IMAGE_F_SHA256                   0x00000002 /* Hash TLV is present */
```

Comment:
Asking for trouble here. What about making these "reserved, must be zero" instead?

Reply:
I see you plan on handling all the renumbering by changing the magic later.

Reply:
It still leaves a window when the format doesn't match the magic. Another possibility would be to make the magic a third not-to-be-used value. But, is this overkill? It will only affect someone doing a bisect, since these patches will all need to go in together.

Short of changing the magic to completely different values at each patch, it still isn't ever quite right.